### PR TITLE
Remove unused components

### PR DESCRIPTION
**Pull request by pkleybolte**
# Short Description
We have a lot of Pages and Components that have PoC quality and never went live. 
In this branch I removed pages that never went live and were not touched for a long time:
account (last change 8 months ago)
calendar (last change 9 months ago)
courses (last change 5 months ago)

I also removed components that were used exclusively by these pages.

I also commented smelly code to point out that it is bad practice and should not be used as a blueprint for new components.

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/organisms/LernstoreCollectionDetailView.vue [hunk 1]
```diff
@@ -267,7 +267,7 @@ export default {
 			try {
 				// Clears the previous collection elements before rendering the new ones
 				this.$store.commit("content/clearElements");
-
+				// TODO wrong use of store (not so bad)
```
Comment (dominikbruechner) [inline src/components/organisms/LernstoreCollectionDetailView.vue:270]: in the 'not so bad' cases you are hinting that we need to rewrite error/success logic? move them more globally I guess...

File: src/pages/account/email/edit.vue [hunk 2]
```diff
@@ -1,152 +0,0 @@
-<template>
```
Comment (dominikbruechner) [inline src/pages/account/email/edit.vue:1]: The account pages were written quite recently, right? (After all I am agreeing to remove them, as we are not going to release them soon...)

File: src/pages/calendar/index.vue [hunk 3]
```diff
@@ -1,321 +0,0 @@
...
-				:header="header"
-				:config="config"
-				@day-click="handleDateClick"
-				@event-selected="eventClick"
-			></full-calendar>
-		</div>
-	</div>
-</template>
-<script>
-import "fullcalendar/dist/fullcalendar.css";
```
Comment (dominikbruechner) [inline src/pages/calendar/index.vue:52]: here the fullcalendar package needs to be removed too

File: src/pages/account/email/edit.vue (hunk 2 above)
Comment (pkleybolte) [reply src/pages/account/email/edit.vue:1]: According to the latest changes the account pages were not touched in 8 months. The courses are actually "younger".

**Review by dominikbruechner — APPROVED** [review]

**Comment by pkleybolte** [comment]
@schul-cloud-bot update screenshots

**Review by dominikbruechner — APPROVED** [review]

**Pull request merged**